app/assets/sass/main.scss

```diff
@@ -5,5 +5,10 @@
 @import 'components/related-nav';
 
 ///////////////////////////////////////////
-// Add your custom CSS/Sass styles below... 
+// Add your custom CSS/Sass styles below...
 ///////////////////////////////////////////
+
+// reset visited link colour
```

Comment:
Is this something that we want for everyone?

If so, needs new line at end of file

Reply:
Moved this to `nhsuk.css` and added new line.